tests/lib/rules/jsx-wrap-multilines.js

```diff
@@ -501,7 +525,7 @@ const ATTR_PAREN_NEW_LINE_AUTOFIX_FRAGMENT = `
 `;
 
 function addNewLineSymbols(code) {
-  return code.replace(/\(</g, '(\n<').replace(/>\)/g, '>\n)');
+  return code.replace(/\((\s*)</g, '($1\n<').replace(/>(\s*)\)/g, '>\n$1)');
```

Comment:
i know this code is already in master, but this makes the test cases very inscrutable :-/

i feel like i'd much rather see an explicit `output` example for each input.

Reply:
Yeah, me too. I can add explicit output for the tests touched by this PR.